components/Cta/CtaLink.tsx

```diff
@@ -47,6 +47,10 @@ export const CtaLink = React.forwardRef<HTMLAnchorElement, CtaLinkProps>(
     if (isInternal) {
       myLink = cachedUrl || href;
 
+      if (myLink === 'home') {
+        myLink = '';
```

Comment:
Shouldn't this be `/` instead of an empty string?

Reply:
It could be either since the conditional right below it adds the `/ `if it doesn't start with one already 😄 
I didn't expect a review this fast 😂